common/components/componentsBySection/FindEvents/EventCard.jsx

```diff
@@ -71,12 +72,33 @@ class EventCard extends React.PureComponent<Props> {
   }
 
   _renderEventTime(): React$Node {
-    const Event: EventTileAPIData = this.props.event;
+    const event: EventTileAPIData = this.props.event;
+    return (
+      <div className="EventCard-time">
+        {event.event_date_start && moment(event.event_date_start) > moment() && (
+          <h2>
+            <Moment format="LT">{event.event_date_start}</Moment>
+          </h2>
+        )}
+      </div>
+    );
+  }
+
+  _renderStartEndDateTime(): React$Node {
+    const event: EventTileAPIData = this.props.event;
+    const dateTimeFormat: string = "LT MMM. D";
     return (
       <div className="EventCard-time">
-        {Event.event_date_start && moment(Event.event_date_start) > moment() && (
+        {event.event_date_start && moment(event.event_date_start) > moment() && (
           <h2>
-            <Moment format="LT">{Event.event_date_start}</Moment>
+            <p>
+              Begins:<br/>
+              <Moment format={dateTimeFormat}>{event.event_date_start}</Moment>
+            </p>
+            <p>
+              Ends:<br/>
+              <Moment format={dateTimeFormat}>{event.event_date_end}</Moment>
+            </p>
```

Comment:
If we remove the containing H2, we can avoid <br> and also style the headers and dates differently. 

I think giving "Begins:" and "Ends:" the same font size/weight (\<h2>) as the event title makes the card visually lopsided with all the added date information there, so I'd suggest this:

```
<h3>Begins:</h3>
<p><Moment format .... </Moment></p>
<h3>Ends:</h3>
<p><Moment format .... </Moment></p>
```

and in order to fix some spacing issues that come up here,
https://github.com/DemocracyLab/CivicTechExchange/blob/master/civictechprojects/static/css/partials/_FindEventController.scss#L31 change this to `.EventCard-root h2, .EventCard-root h3`


And then to avoid an issue with long location strings, like this:
![image](https://user-images.githubusercontent.com/16870466/205376533-df12e1f3-b6a5-409d-b8d1-b12c620ce042.png)

https://github.com/DemocracyLab/CivicTechExchange/blob/master/civictechprojects/static/css/partials/_FindEventController.scss#L109-L112
add `margin-right: 0.5rem;` here



the end result should be this:
![image](https://user-images.githubusercontent.com/16870466/205376828-ed410607-eca3-4f03-b979-72ec18ff954d.png)